Makefile

```diff
@@ -114,521 +105,312 @@ endef
 PHONY: # Since .PHONY reads % as a literal %, we need this target as a prereq to treat pattern targets as .PHONY
-PHONY_TARGETS += all all-clean build-and-check clean $(addprefix CLEAN_,$(CLEAN_FILES)) extract build patch check expected
-PHONY_TARGETS += dump-disk $(addprefix dump-disk_,eu hk jp10 jp11 saturn us usproto) extract-disk disk disk-prepare disk-debug
-PHONY_TARGETS += format-src format-src.run $(addprefix FORMAT_,$(FORMAT_SRC_FILES)) format-tools $(addprefix FORMAT_,$(PY_TOOLS_DIRS))
-PHONY_TARGETS += format-symbols format-symbols.run $(addprefix format-symbols_,us hd pspeu saturn) $(addprefix FORMAT_,$(FORMAT_SYMBOLS_FILES)) format-license
-PHONY_TARGETS += force-symbols $(addprefix FORCE_,$(FORCE_SYMBOLS)) force-extract context function-finder duplicates-report
-PHONY_TARGETS += git-submodules update-dependencies update-dependencies-all $(addprefix dependencies_,us pspeu hd saturn) requirements-python graphviz
+PHONY_TARGETS += all extract build patch check 
+PHONY_TARGETS += dump-disk $(addprefix dump-disk.,eu hk jp10 jp11 saturn us usproto) extract-disk
+PHONY_TARGETS += format-src format-src.run $(addsuffix .format-src,$(FORMAT_SRC_FILES)) format-tools format-symbols format-license
+PHONY_TARGETS += $(addsuffix .force-symbols,$(notdir $(wildcard $(BUILD_DIR:$(VERSION)=us)/*.elf))) force-extract context $(addprefix format-symbols.,us hd pspeu saturn $(FORMAT_SYMBOLS_FILES))
+PHONY_TARGETS += git-submodules update-dependencies update-dependencies-all $(addprefix dependencies.,us pspeu hd saturn) python-dependencies graphviz $(DOSEMU_APP)
 PHONY_TARGETS += help get-debug get-phony get-silent
-MUFFLED_TARGETS += $(PHONY_TARGETS) $(MASPSX_APP) $(MWCCGAP_APP) $(MWCCPSP) $(SATURN_SPLITTER_DIR) $(SATURN_SPLITTER_APP) $(EXTRACTED_DISK_DIR)
-MUFFLED_TARGETS += $(DOSEMU_APP) $(ASMDIFFER) $(dir $(M2C_APP)) $(M2C_APP) $(PERMUTER_APP) $(SOTNDISK) $(SOTNASSETS) $(VENV_DIR) $(VENV_DIR)/bin
-.PHONY: $(PHONY_TARGETS)
-# Specifying .SILENT in this manner allows us to set the DEBUG environment variable and display everything for debugging
-#$(DEBUG).SILENT: $(MUFFLED_TARGETS)
-# These are walls of text, so they're redirected to files instead of stdout for debugging
-get-debug: get-phony get-silent
-get-phony:
-	echo ".PHONY:" > make.phony.targets
-	$(foreach target,$(PHONY_TARGETS),echo $(target) >> make.phony.targets;)
-get-silent:
-	echo ".SILENT:" > make.silent.targets
-	$(foreach target,$(MUFFLED_TARGETS),echo $(target) >> make.silent.targets;)
+MUFFLED_TARGETS += $(PHONY_TARGETS) $(MASPSX_APP) $(MWCCGAP_APP) $(MWCCPSP) $(SATURN_SPLITTER_DIR) $(SATURN_SPLITTER_APP) $(EXTRACTED_DISK_DIR) $(ASMDIFFER_APP) $(PERMUTER_APP) $(dir $(M2C_APP)) $(M2C_APP)
+MUFFLED_TARGETS += $(DOSEMU_DIR) tools/dosemu.make.chkpt tools/python-dependencies.make.chkpt tools/graphviz.make.chkpt $(SOTNDISK) $(SOTNASSETS) $(VENV_DIR) $(VENV_DIR)
+.PHONY: $(PHONY_TARGETS) expected clean force-symbols disk disk-prepare disk-debug
```

Comment:
I began looking at targets to see where I can trim all of this down.  I was able to move some targets directly to .PHONY and clean it up a bit, but I only did it in a few select places for now.